__tests__/app.test.js

```diff
@@ -148,109 +148,245 @@ describe("/api/reviews/:review_id/comments", () => {
+        expect(reviews.length).toBe(13);
+        reviews.forEach((element) => {
+          expect(element).toMatchObject({
+            owner: expect.any(String),
+            title: expect.any(String),
+            review_id: expect.any(Number),
+            category: expect.any(String),
+            review_img_url: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            designer: expect.any(String),
+            comment_count: expect.any(Number),
+          });
+        });
+        expect(reviews).toBeSortedBy("comment_count", { descending: false });
+      });
+  });
+  test("GET 200 - responds with array of reviews sorted by designer ascending (defaulted)", () => {
+    return request(app)
+      .get("/api/reviews?sort_by=designer&order=asc")
+      .expect(200)
+      .then((res) => {
+        const reviews = res.body.reviews;
+        expect(reviews.length).toBe(13);
+        reviews.forEach((element) => {
+          expect(element).toMatchObject({
+            owner: expect.any(String),
+            title: expect.any(String),
+            review_id: expect.any(Number),
+            category: expect.any(String),
+            review_img_url: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            designer: expect.any(String),
+            comment_count: expect.any(Number),
+          });
+        });
+        expect(reviews).toBeSortedBy("designer", { descending: false });
+      });
+  });
+  test("GET 200 - responds with array of reviews filtered by category social deduction in ascending date order", () => {
+    return request(app)
+      .get("/api/reviews?category=social deduction&order=asc")
+      .expect(200)
+      .then((res) => {
+        const reviews = res.body.reviews;
+        expect(reviews.length > 0).toBe(true);
+        reviews.forEach((element) => {
+          expect(element).toMatchObject({
+            owner: expect.any(String),
+            title: expect.any(String),
+            review_id: expect.any(Number),
+            category: "social deduction",
+            review_img_url: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            designer: expect.any(String),
+            comment_count: expect.any(Number),
+          });
+          expect(reviews).toBeSortedBy("created_at", { descending: false });
+        });
+      });
+  });
+  test("GET 400 - bad order query", () => {
+    return request(app)
+      .get("/api/reviews?order=badString")
+      .expect(400)
+      .then((res) => {
+        expect(res.body.msg).toBe("Invalid order query");
+      });
+  });
+  test("GET 400 - bad order query", () => {
+    return request(app)
+      .get("/api/reviews?sort_by=badString")
+      .expect(400)
+      .then((res) => {
+        expect(res.body.msg).toBe("Invalid sort_by query");
+      });
```

Comment:
Need a few more tests here. 

- What if the category is invalid for example `category=1234` 
- What if the category is valid but doesn't exist in db? eg. `category=banana`
- What if the category exists but has no reviews associated with it?